src/main/resources/data/minecraft/tags/blocks/flower_pots.json

```diff
@@ -0,0 +1,7 @@
+{
+	"values": [
+		"crimson:potted_crimson_sapling",
```

Comment:
again here

Reply:
also don't you need replace=false or something

Reply:
Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.